src/DynamoCoreWpf/Controls/IncanvasSearchControl.xaml.cs

```diff
@@ -31,7 +31,7 @@ public SearchViewModel ViewModel
         public InCanvasSearchControl()
         {
             InitializeComponent();
-            if (Application.Current != null)
+            if(string.IsNullOrEmpty(ViewModel.dynamoViewModel.Model.HostAnalyticsInfo.HostName))
             {
                 Application.Current.Deactivated += CurrentApplicationDeactivated;
```

Comment:
So without the check above, this line could crash. I think we still need to check the null or use ? to prevent crash

Reply:
`Application.Current` is still being checked.
But I am confused by this logic, under what circumstances would `HostName` be null or empty string?

How can you guarantee that all DynamoSandbox implementations have null or empty string for host name and those in hosts as addins do not?

Reply:
Are there any specific integrators that you would like me to confirm, as of now I can confirm C3D, Revit, Alias is setting a HostName.